test/gtest/ucs/test_datatype.cc

```diff
@@ -261,6 +270,52 @@ UCS_TEST_F(test_datatype, hlist_for_each_extract_if) {
     EXPECT_TRUE(v_elems.empty());
 }
 
+UCS_TEST_F(test_datatype, hlist_foreach_safe) {
+    for (int length : {0, 1, 1, 2, 3, 4, 5, 10, 100, 1000}) {
```

Comment:
detail: may only one `1,`?

Reply:
because of random in line 300 wanted to test length=1 twice